app/models/miq_report/generator.rb

```diff
@@ -88,6 +88,34 @@ def get_include
     include_as_hash(include.presence || invent_report_includes)
   end
 
+  def polymorphic_includes
+    @polymorphic_includes ||= begin
+      polymorphic_rels = []
+      top_level_rels   = []
+      top_level_rels  += Array(get_include.try(:keys))
+      top_level_rels  += Array(get_include_for_find.try(:keys))
+
+      top_level_rels.uniq.each do |assoc|
+        reflection = db_klass.reflect_on_association(assoc)
+        polymorphic_rels << assoc if reflection && reflection.polymorphic?
+      end
```

Comment:
Note:  This is basically a reworking of the code that was removed here:

- https://github.com/ManageIQ/manageiq/commit/8cc2277b
- https://github.com/ManageIQ/manageiq/commit/8cc2277b#diff-8d948055de14ced0e63abf9637a9a788L438-L443